x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/overview/overview.tsx

```diff
@@ -33,6 +40,20 @@ export const Overview = () => {
   } = useMetadataStateContext();
   const { metrics } = useDataViewsContext();
   const isFullPageView = renderMode.mode === 'page';
+  const { dataStreams, status: dataStreamsStatus } = useAssetEntitySummary({
+    entityType: asset.type,
+    entityId: asset.type === 'host' ? asset.name : asset.id,
+  });
+  const addMetricsCalloutId: AddMetricsCalloutKey =
+    asset.type === 'host' ? 'hostOverview' : 'containerOverview';
+  const [dismissedAddMetricsCallout, setDismissedAddMetricsCallout] = useLocalStorage(
+    `infra.dismissedAddMetricsCallout.${addMetricsCalloutId}`,
+    false
+  );
+  const isDockerContainer = useIntegrationCheck({ dependsOn: INTEGRATIONS.docker });
+  const isKubernetesContainer = useIntegrationCheck({
+    dependsOn: INTEGRATIONS.kubernetesContainer,
+  });
```

Comment:
The typical use of `useIntegrationCheck` is to determine whether it belongs to a specific integration, not to check if it belongs to more than one.

I don't think we should modify the function just because we need to check for either of the two integrations on a single occasion.

Reply:
> The typical use of useIntegrationCheck is to determine whether it belongs to a specific integration, not to check if it belongs to more than one

Makes sense. However, there's nothing fundamentally wrong with making check multiple integrations. The hook's purpose is to check if metadata returns certain fields.

Reply:
I'm not sure I'll make the change in this PR as this was a nit, but I'm curious to know how you'd approach it.

If we pass a list in `dependsOn`, should the response still be a boolean, or should it be an object where each string we passed is a key with the boolean as its value?

I mention this because if we keep the response as a simple boolean, I don't quite see the point of checking whether the metadata includes at least one of the strings from `dependsOn`. It doesn't seem like it would provide truly relevant information for its use.

Reply:
It's a nit. 

For this purpose, and what we currently need, I would implement it as all-or-nothing. Either metadata has all in `dependsOn` and returns `true`, otherwise `false`. 

Originally, when the asset details briefly supported nginx charts for Hosts, the code responsible for checking it did exactly that. Two integrations were mandatory to show the charts: `nginx.acesss` and `nginx.stubstatus`